src/colorizer/utils/react_utils.ts

```diff
@@ -289,22 +292,44 @@ export type AnnotationState = {
 export const useAnnotations = (): AnnotationState => {
   const annotationData = useConstructor(() => new AnnotationData());
 
-  const [currentLabelIdx, setCurrentLabelIdx] = useState<number | null>(null);
+  const [currentLabelIdx, _setCurrentLabelIdx] = useState<number | null>(null);
   const [isAnnotationEnabled, _setIsAnnotationEnabled] = useState<boolean>(false);
   const [visible, _setVisibility] = useState<boolean>(false);
 
   const [baseSelectionMode, setBaseSelectionMode] = useState<AnnotationSelectionMode>(AnnotationSelectionMode.TIME);
 
   const isSelectRangeHotkeyPressed = useShortcutKey("Shift");
+  const isReuseValueHotkeyPressed = useShortcutKey("Control");
+
+  /** 
+   * The ID of the last clicked object. `null` if the user clicked on the
+   * background. 
+   */
   const [lastClickedId, setLastClickedId] = useState<number | null>(null);
+  /**
+   * The last range of IDs that were edited, used for range-related operations.
+   * If a user clicks on an object again that is one of the endpoints of this
+   * range, any operations will be applied to the entire range. Cleared when
+   * another object is clicked.
+  */
   const [lastEditedRange, setLastEditedRange] = useState<number[] | null>(null);
+  /** 
+   * The range of values that should currently be editable. Value is non-null
+   * when a user interacts with an annotation in order to edit it.
+   */
+  const [activeEditRange, setActiveEditRange] = useState<number[] | null>(null);
 
   // When in time mode, allow hotkeys to temporarily change to range mode.
   let selectionMode = baseSelectionMode;
-  if (baseSelectionMode === AnnotationSelectionMode.TIME && isSelectRangeHotkeyPressed) {
+  if (isSelectRangeHotkeyPressed) {
```

Comment:
Tiny change; pressing the hotkey in both time and track range selection modes temporarily changes it to the range selection mode.